airbyte-cdk/python/airbyte_cdk/sources/file_based/config/abstract_file_based_spec.py

```diff
@@ -34,6 +53,21 @@ class AbstractFileBasedSpec(BaseModel):
         order=10,
     )
 
+    use_file_transfer: bool = Field(
+        title="File Sync (Experimental)", description="Enable file-based bulk load", default=False, airbyte_hidden=True
+    )
+
```

Comment:
Should we drop this boolean in favor of the new `delivery_type` input?

Reply:
Yes!

Reply:
Removed.